src/olympia/users/forms.py

```diff
@@ -210,7 +210,11 @@ def save(self, log_for_developer=True):
             tasks.resize_photo.delay(tmp_destination, u.picture_path,
                                      set_modified_on=[u])
 
-        for (i, n) in notifications.NOTIFICATIONS_BY_ID.items():
+        visible_notifications = (
+            notifications.NOTIFICATIONS_BY_ID if self.instance.is_developer
+            else notifications.NOTIFICATIONS_BY_ID_NOT_DEV)
+
+        for (i, n) in visible_notifications.items():
```

Comment:
I know that this is old code but can we rename `n` and `i` to something more proper lilke `notification` and `notification_id`?